src/Shared/Cosmos/CosmosConstants.cs

```diff
@@ -5,6 +5,12 @@ namespace Aspire.Hosting.Azure.Cosmos;
 
 internal static class CosmosConstants
 {
+    /// <summary>
+    /// Defines the application name used to interact with the aszure cosmos db. This will be suffixed to the
```

Comment:
Typo: aszure

Reply:
since it's a brand we should probably fix in main

Reply:
```suggestion
    /// Defines the application name used to interact with the Azure cosmos db. This will be suffixed to the
```